hw2/Index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>hw2</title>
+	<link rel="stylesheet" href="css/reset.css">
+	<link rel="stylesheet" href="css/fonts.css">
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+	<div class="wrapper">
+		<header>
+			<img src="img/logo.png">
+			<ul class="top-menu">
+				<li><a href="#">Главная</a></li>
+				<li><a href="#">Товары</a></li>
+				<li><a href="#">О_Нас</a></li>
+				<li><a href="#">Контакты</a></li>
+			</ul>	
+		</header>
+		<section class="container">
+			<div class="left-menu">
+				<ul>
+					<li><a href="#">Ноутбуки</a></li>
+					<li><a href="#">Планшеты</a></li>
+					<li><a href="#">Телефоны</a></li>
+					<li><a href="#">Телевизоры</a></li>
+					<li><a href="#">Бытовая техника</a></li>
+					<li><a href="#">Автотовары</a></li>
+				</ul>
+			</div>	
+			<div class="content">
+				<div class="image">
+					<img src="img/image.png">
+				</div>
+				<article class="article">
+					Lorem Ipsum - це текст-"риба", що використовується в друкарстві та дизайні. Lorem Ipsum є, фактично, стандартною "рибою" аж з XVI сторіччя, коли невідомий друкар взяв шрифтову гранку та склав на ній підбірку зразків шрифтів. "Риба" не тільки успішно пережила п'ять століть, але й прижилася в електронному верстуванні, залишаючись по суті незмінною. Вона популяризувалась в 60-их роках минулого сторіччя завдяки виданню зразків шрифтів Letraset, які містили уривки з Lorem Ipsum, і вдруге - нещодавно завдяки програмам комп'ютерного верстування на кшталт Aldus Pagemaker, які використовували різні версії Lorem Ipsum.
+				</article>
+			</div>		
+		</section>
+		<ul class="footer">
+			<li><a href="#">Главная</a></li>
+			<li><a href="#">Товары</a></li>
+			<li><a href="#">О_Нас</a></li>
+			<li><a href="#">Контакты</a></li>
+			<li class="copyright">2014 © All rights reserved</li>
```

Comment:
I've made this mistake when I was doing this HW. Copyright is not a logical part of this list. So split it.